ui-speedspacechart/src/__tests__/utils.spec.ts

```diff
@@ -60,7 +61,8 @@ describe('getGraphOffsets', () => {
   it('should return the correct width and height offsets', () => {
     const width = 100;
     const height = 200;
-    const { WIDTH_OFFSET, HEIGHT_OFFSET } = getGraphOffsets(width, height);
+    const declivities = true;
+    const { WIDTH_OFFSET, HEIGHT_OFFSET } = getGraphOffsets(width, height, declivities);
     expect(WIDTH_OFFSET).toBe(40);
     expect(HEIGHT_OFFSET).toBe(120);
```

Comment:
As I mentioned below, don't forget to run the tests locally using npm run test until we have CI set up to do it for us. I have added 3 cases : with declivities set to true, false and undefined.
```typescript
describe('getGraphOffsets', () => {
  const width = 200;
  const height = 150;

  it('should return correct width and height offsets when declivities is true', () => {
    const result = getGraphOffsets(width, height, true);

    expect(result).toEqual({
      WIDTH_OFFSET: 98,
      HEIGHT_OFFSET: 70,
    });
  });

  it('should return correct width and height offsets when declivities is false', () => {
    const result = getGraphOffsets(width, height, false);

    expect(result).toEqual({
      WIDTH_OFFSET: 140,
      HEIGHT_OFFSET: 70,
    });
  });

  it('should return correct width and height offsets when declivities is undefined', () => {
    const result = getGraphOffsets(width, height);

    expect(result).toEqual({
      WIDTH_OFFSET: 140,
      HEIGHT_OFFSET: 70,
    });
  });
});
```

Reply:
Done !